sycl/ur_win_proxy_loader/ur_win_proxy_loader.cpp

```diff
@@ -158,41 +156,7 @@ void preloadLibraries() {
 
 /// windows_pi.cpp:loadOsPluginLibrary() calls this to get the DLL loaded
 /// earlier.
-__declspec(dllexport) void *getPreloadedPlugin(
-    const std::filesystem::path &PluginPath) {
-
-  MapT &dllMap = getDllMap();
-
-  // All entries in the dllMap have the same parent directory.
-  // To avoid case sensivity issues, we don't want to do string comparison but
-  // just make sure that directory of the entires in the map and directory of
-  // the PluginPath are equivalent (point to the same physical location).
-  auto match = dllMap.end();
-  std::error_code ec;
-  if (!dllMap.empty() &&
-      std::filesystem::equivalent((dllMap.begin())->first.parent_path(),
-                                  PluginPath.parent_path(), ec)) {
-    // Now we can search only by filename. Result might be nullptr (not found),
-    // which is perfectly valid.
-    match =
-        std::find_if(dllMap.begin(), dllMap.end(),
-                     [&](const std::pair<std::filesystem::path, void *> &v) {
-                       return v.first.filename() == PluginPath.filename();
-                     });
-  }
-
-  if (match == dllMap.end()) {
-    // unit testing? return nullptr (not found) rather than risk asserting below
-    if (PluginPath.string().find("unittests") != std::string::npos)
-      return nullptr;
-
-    // Otherwise, asking for something we don't know about at all, is an issue.
-    std::cout << "unknown plugin: " << PluginPath << std::endl;
-    assert(false && "getPreloadedPlugin was given an unknown plugin path.");
-    return nullptr;
-  }
-  return match->second;
-}
+__declspec(dllexport) void *getPreloadedURLib() { return getDllHandle(); }
```

Comment:
just below here, at line 166, I notice that DllMain is still checking for `SYCL_PI_TRACE`.  But didn't all y'all remove that environment variable and swtich to `SYCL_UR_TRACE` or some such?  

Now might be a good time to update that.

Reply:
Fixed now, good catch